monitoring/kaggle/scripts/test-kaggle-integration.sh

```
@@ -0,0 +1,74 @@
+set -e
+
+echo "🔍 Testing Kaggle API Integration..."
+
+# Colors
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+
+# Load environment variables
+if [ -f .env ]; then
+    export $(cat .env | grep -v '^#' | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)